Campaigns/apt sofacy.txt

```diff
@@ -0,0 +1,7 @@
+// Original Sigma Rule: https://github.com/Neo23x0/sigma/blob/master/rules/apt/apt_sofacy.yml
+// Questions via Twitter: @janvonkirchheim
+ProcessCreationEvents
+| where EventTime > ago(7d)
+| where ProcessCommandLine matches regex @'rundll32.exe %APPDATA%\*.dat",*'
```

Comment:
I'm not sure if this regex is logically correct, since `*` has a different meaning in RegEx - so I think it won't capture anything.
Can you try it against real data, see if the query catches anything?

Reply:
looks like its working in general ...:

![2019-06-19 11_36_00-Clipboard](https://user-images.githubusercontent.com/6720053/59754588-ac917e80-9286-11e9-8bfd-e698f309d24e.png)

Reply:
But the meaning of this regex is:
find a string that contains "conhost.exe" followed by zero or more "e" characters.

So I say the current query has is a logical bug.
@'rundll32.exe %APPDATA%\*.dat",*'

This one translates to:
Look for the string "rundll32.exe %APPDATA%\" followed by zero or more `"` characters, followed by any character (that's the dot) followed by `dat"," followed by zero or more `,` characters.

Reply:
you are right. I changed * to .*